lxd/device/nic_ovn.go

```diff
@@ -554,45 +554,18 @@ func (d *nicOVN) Start() (*deviceConfig.RunConfig, error) {
 	// Populate device config with volatile fields if needed.
 	networkVethFillFromVolatile(d.config, saveData)
 
-	v := d.volatileGet()
-
-	// Retrieve any last state IPs from volatile and pass them to OVN driver for potential use with sticky
-	// DHCPv4 allocations.
-	var lastStateIPs []net.IP
-	for _, ipStr := range shared.SplitNTrimSpace(v["last_state.ip_addresses"], ",", -1, true) {
-		lastStateIP := net.ParseIP(ipStr)
-		if lastStateIP != nil {
-			lastStateIPs = append(lastStateIPs, lastStateIP)
-		}
-	}
-
 	// Add new OVN logical switch port for instance.
-	logicalPortName, dnsIPs, err := d.network.InstanceDevicePortStart(&network.OVNInstanceNICSetupOpts{
+	logicalPortName, _, err := d.network.InstanceDevicePortStart(&network.OVNInstanceNICSetupOpts{
```

Comment:
Worth getting rid of the returned `dnsIPs` argument from `InstanceDevicePortStart()`?

This now looks to be not used anymore.

Reply:
Nvm, also done in one of the next commits.